### DISCUSSION TITLE
Browser not secure

### DISCUSSION THREAD

**Discussion by learning277** [community/community#64347] [New to GitHub]
Downloaded mull browser is saying not secure.

**Comment by queenofcorgis** [comment]
Hi @learning277 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Mull. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by RamtinKosari** [comment]
Hi @learning277 
If you downloaded a browser called "Mull" and it is displaying a "Not Secure" message, it could indicate a potential security issue with the browser or the website you are visiting. Here are a few things you can consider:

Verify the source: Ensure that you downloaded the browser from a trusted and official source. Stick to well-known and reputable websites or app stores to download browsers to minimize the risk of downloading a malicious or insecure version.

Update the browser: Check if there are any available updates for the Mull browser and install them. Updates often include security patches that address known vulnerabilities.

Check the website's security: If the "Not Secure" message appears when visiting a specific website, it indicates that the website does not have a valid SSL certificate or is not using HTTPS encryption. Exercise caution when entering sensitive information on such sites, especially passwords or personal data.

Use a different browser: If you are concerned about the security of the Mull browser, consider using a different well-established and trusted browser like Google Chrome, Mozilla Firefox, or Microsoft Edge. These browsers have robust security measures in place and are regularly updated.

Remember to always exercise caution when browsing the internet and ensure that your chosen browser and the websites you visit prioritize security.
